examples/pii_detection_presidio.py

```diff
@@ -0,0 +1,161 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC # PII detection with DiscoverX & Presidio
+# MAGIC
+# MAGIC This notebooks uses [DiscoverX](https://github.com/databrickslabs/discoverx) to run PII detection with [Presidio](https://microsoft.github.io/presidio/) over a set of tables in Unity Catalog.
```

Comment:
Would a comment about the value DiscoverX adds to this make sense here? I guess, at the moment, DiscoverX only helps with the initial data-prep, i.e. selecting all relevant columns from all tables corresponding to the specified wildcards. Is that correct? Should we add this to the intro section of the notebook?